src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.spec.ts

```diff
@@ -70,6 +72,9 @@ describe('StripeUPEPaymentStrategy', () => {
         storeCreditActionCreator = new StoreCreditActionCreator(
             new StoreCreditRequestSender(requestSender)
         );
+
+        remoteCheckoutRequestSender = new RemoteCheckoutRequestSender(requestSender);
```

Comment:
I think you should add a Test case where `cancelToken` happens on `deinitialize`